codeHF/config_tasks.sh

```diff
@@ -150,8 +164,7 @@ function MakeScriptO2 {
   [ $DOO2_PID_TOF -eq 1 ] && WORKFLOWS+=" o2-analysis-pid-tof-full"
   [ $DOO2_PID_TOF_QA -eq 1 ] && WORKFLOWS+=" o2-analysis-pid-tof-qa-mc"
   # Vertexing
-  [ $DOO2_SKIM -eq 1 ] && WORKFLOWS+=" o2-analysis-hf-track-index-skims-creator"
-  [ $DOO2_SKIM_V0 -eq 1 ] && WORKFLOWS+=" o2-analysis-hf-track-index-skims-creator_v0"
+  [ $DOO2_SKIM -eq 1 ] && WORKFLOWS+=" o2-analysis-hf-track-index-skims-creator${EVSELSUFFIX}${CASCADESUFFIX}"
```

Comment:
Maybe better to define a variable here like `WF_SKIM="o2-analysis-hf-track-index-skims-creator${EVSELSUFFIX}${CASCADESUFFIX}"`
and use it later with `ReplaceString`

Reply:
Done!